spec/controllers/two_factor_authentication/totp_verification_controller_spec.rb

```diff
@@ -59,6 +59,27 @@
       end
     end
 
+    context 'with multiple TOTP configurations' do
+      it 'allows using codes generated from any registered TOTP configuration' do
+        user = create(:user, :fully_registered)
+        app1 = Db::AuthAppConfiguration.create(user, user.generate_totp_secret, nil, 'foo')
+        app2 = Db::AuthAppConfiguration.create(user, user.generate_totp_secret, nil, 'bar')
```

Comment:
these variables are unused, can we drop the assignemnts?
```suggestion
        Db::AuthAppConfiguration.create(user, user.generate_totp_secret, nil, 'foo')
        Db::AuthAppConfiguration.create(user, user.generate_totp_secret, nil, 'bar')
```

Reply:
They are used on lines 72 and 78

Reply:
ah my bad, thx!